static/js/recent_topics.js

```diff
@@ -25,10 +25,12 @@ let row_focus = 0;
 let col_focus = 1;
 
 // The number of selectable actions in a recent_topics.  Used to
-// implement wraparound of elements with the right/left keys.  Must be
-// increased when we add new actions, or rethought if we add optional
-// actions that only appear in some rows.
-const MAX_SELECTABLE_COLS = 4;
+// implement wraparound of elements with the right/left keys.
+// We may want to make action buttons focusable in future. It is
+// not done because there will always be confusion that if action
+// buttons should be shown for (keyboard) focused table row or (mouse) hovered
+// table row.
+const MAX_SELECTABLE_COLS = 2;
```

Comment:
After playing with this, I feel like having it share space kinda messes up the keyboard navigation.  And I don't like being unable to see the timestamp for the thing I'm looking at.  What do you think about just having these elements by in the `topic` column right-aligned?  That would save space (since most of the space these icons take is their heading at present).

Reply:
(I also don't like losing access to the exact time as you do in a later commit).